apps/storybook/src/lib/blockscout/fixture.ts

```diff
@@ -434,3 +434,5 @@ export const TXN_VITALIK_TRANSFER = {
 	block_number: 21067426,
 	has_error_in_internal_transactions: false,
 };
+
+export const TXN_LIST = [TXN_VITALIK_DEPOSIT, TXN_VITALIK_TRANSFER];
```

Comment:
@metalboyrick can help to add more fixtures? 
picking
- erc20 transfer

generating  createTxnFixture()
- 100 records, for perf loading test

Reply:
can ser no problem